guides/schematics.md

```diff
@@ -0,0 +1,53 @@
+Angular Material comes packaged with Angular CLI schematics to make
+creating new components easy.
+
+## Install Schematics
+Schematics come packaged with Angular Material, so once you have
+installed the npm package, they will be available via the Angular CLI.
+
+## Packaged Schematics
+Angular Material has 4 schematics it comes packaged with:
+
+- Material Shell
+- Navigation
+- Dashboard
+- Table
+
+### Shell Schematic
+The shell schematic will help you quickly add Material to a new project. 
+This schematic will:
+
+- Ensure project depedencies in `package.json`
+- Ensure project depedencies in your app module
+- Add Prebuilt or Setup Custom Theme
```

Comment:
Should a note be included on how to choose a prebuilt or custom theme?

Reply:
It does it automatically.